packages/wonder-blocks-link/src/components/__tests__/link.test.tsx

```diff
@@ -436,4 +436,49 @@ describe("Link", () => {
             expect(link).toHaveStyle(`outline: 1px solid ${Color.white}`);
         });
     });
+
+    describe("external link", () => {
+        test("target attribute passed down correctly", () => {
+            // Arrange
+            render(
+                <Link href="/" target="_blank">
+                    Click me!
+                </Link>,
+            );
+
+            // Act
+            const link = screen.getByText("Click me!");
+
+            // Assert
+            expect(link).toHaveAttribute("target", "_blank");
+        });
+
+        test("render icon when `target=_blank`", () => {
+            // Arrange
+            render(
+                <Link href="/" target="_blank">
+                    Click me!
+                </Link>,
+            );
+
+            // Act
+            const link = screen.getByText("Click me!");
+            const icon = screen.getByTestId("external-icon");
+
+            // Assert
+            expect(link.innerHTML).toEqual(expect.stringContaining("<svg"));
+            expect(icon).toBeInTheDocument();
+        });
+
+        test("does not render icon", () => {
```

Comment:
**suggestion:** Test case names should indicate an outcome and the condition that leads to that outcome, like a requirement. For example, in this case, "does not render icon when there is no target".

This helps folks who encounter a test failure understand the intention of the test; that under X circumstances we should expect Y behaviour.